### DISCUSSION TITLE
What I Love About Go/Baduk - Kiran Gangadharan

### DISCUSSION THREAD

**Discussion by giscus** [kirang89/kirang.in#3] [General]
# What I Love About Go/Baduk - Kiran Gangadharan

I hadn’t thought about Go or even played it until recently, when a colleague of mine who worked at nilenso raved about the elegance of the game. He mentioned Cho Chikun’s book to be an excellent introduction when I enquired further. Given that we had the book in our library, I decided to give it a read. By the end of it, I was fascinated not just by the beauty and elegance of the game, but also the history and culture.

https://kirang.in/post/what-i-love-about-baduk/

**Comment by kirang89** [comment]
Testing a new comment. Nice post, btw :)

**Comment by mikecook37** [comment]
Choosing a reliable online casino is the key to safe gaming. Licenses, player reviews, and transparent terms and conditions ensure trust and protect against fraud. Players who focus on proven platforms such as https://glassi-casino.com.in/en-in/ can concentrate on strategy and entertainment without being distracted by questionable issues. This makes the gaming process comfortable and safe, allowing you to enjoy yourself without any unnecessary worries.
